backends/bmv2/run-bmv2-test.py

```diff
@@ -40,12 +40,14 @@
 SUCCESS = 0
 FAILURE = 1
 
+
 class Options(object):
     def __init__(self):
         self.binary = ""                # this program's name
         self.cleanupTmp = True          # if false do not remote tmp folder created
         self.p4Filename = ""            # file that is being compiled
         self.compilerSrcDir = ""        # path to compiler source tree
+        self.compilerBuildDir = "build"  # the build subfolder
```

Comment:
should default this based on the current working directory -- if you're running in a subdirectory of the compilerSrcDir, then that should be the build dir.  Maybe do that in main (after setting compilerSrcDir) rather than here?

Reply:
Okay pushed a version that uses the working directory as default.